recipe/menu/open_prompt.bat

```diff
@@ -0,0 +1,10 @@
+:: This is used to initialize the bash prompt on Windows.
+@ECHO OFF
+
+call %SCRIPTS_DIR%\Activate.bat
```

Comment:
Maybe this should be 
```suggestion
call %__PREFIX__%\Scripts\Activate.bat
```
WDYT @hoechenberger ?

Reply:
... based on how `open_prompt.sh` works, I think it should be. I'll commit but feel free to correct me if I'm wrong